packages/widgets/src/tabbar.ts

```diff
@@ -893,6 +914,9 @@ class TabBar<T> extends Widget {
     let ci = this._currentIndex;
     let bh = this.insertBehavior;
 
+
+    // TODO: do we need to do an update to update the aria-selected attribute?
```

Comment:
@afshin points out that the two TODO items should be taken care of. Agreed. In a later version I just deleted the two notes, which may indicate that I thought about it and decided it wasn't necessary. That should be double-checked.